t/mojo/user_agent_tls.t

```diff
@@ -99,4 +100,22 @@ is $ua->ioloop->stream($tx->connection)->handle->get_cipher, 'AES256-SHA',
 is $ua->ioloop->stream($tx->connection)->handle->get_sslversion, 'TLSv1',
   'TLSv1 has been negotiatied';
 
+# Web server with invalid certificate
+$daemon = Mojo::Server::Daemon->new(
+  app    => app,
+  ioloop => Mojo::IOLoop->singleton,
+  silent => 1
+);
+$listen
+  = 'https://127.0.0.1'
+  . '?cert=t/mojo/certs/selfsigned.crt'
+  . '&key=t/mojo/certs/selfsigned.key';
+$port = $daemon->listen([$listen])->start->ports->[0];
```

Comment:
Is this really a new test case? Or is it just doing the same as one of the others?